src/test/resources/suiteHW3.xml

```diff
@@ -0,0 +1,8 @@
+<suite name="suiteHW2" verbose="1" >
+    <test name="Test" >
+        <classes>
+            <class name="com.epam.tc.hw3.scripts.Ex1FromHW2"/>
+            <class name="com.epam.tc.hw3.scripts.Ex2FromHW2"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
Add a new blank line to the end of the file
![image](https://user-images.githubusercontent.com/42721087/210853754-9159897b-59f7-4fc7-96f1-6bbe4840f5c8.png)

Reply:
done